scripts/beaker/epwalsh-stability/llm-360-amber-launch.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -ex
+
+NUM_NODES=4
+
+gantry run \
+  --workspace ai2/OLMo-training  \
+  --task-name llm-360-amber-baseline \
+  --description "LLM 360 Amber 7B in the OLMo codebase" \
+  --priority urgent \
+  --preemptible \
+  --beaker-image petew/olmo-torch23-gantry \
+  --cluster ai2/jupiter-cirrascale-2 \
+  --gpus 8 \
+  --replicas "${NUM_NODES}" \
+  --leader-selection \
+  --host-networking \
+  --budget ai2/oe-training \
+  --no-nfs \
+  --propagate-failure \
+  --synchronized-start-timeout 30m \
+  --no-python \
+  --env LOG_FILTER_TYPE=local_rank0_only \
+  --env OMP_NUM_THREADS=8 \
+  --env OLMO_TASK=model \
+  --env-secret AWS_ACCESS_KEY_ID=DUSTINS_AWS_ACCESS_KEY_ID \
```

Comment:
I know it doesn't really matter, but I've been changing these for my runs (just so things don't suddenly break if someone deletes their key from the shared workspace).